master_changes.md

```diff
@@ -110,6 +111,9 @@ users)
 
 ## Reftests
 ### Tests
+  * Add some related pin tests: fetching, reinstall trigger & simulated pin (deps-only) [#XXX @rjbou]
+  * Add working dir test case to check the case where a requested working-dir package is already pinned with another url [#XXX @rjbou]
+  * Add a pin edit test case to check that the additional information is not lost in case of repin to the same url [#XXX @rjbou]
```

Comment:
```suggestion
  * Add some related pin tests: fetching, reinstall trigger & simulated pin (deps-only) [#6533 @rjbou]
  * Add working dir test case to check the case where a requested working-dir package is already pinned with another url [#6533 @rjbou]
  * Add a pin edit test case to check that the additional information is not lost in case of repin to the same url [#6533 @rjbou]
```